airflow/providers/fab/auth_manager/fab_auth_manager.py

```diff
@@ -263,8 +261,10 @@ def is_authorized_variable(
         return self._is_authorized(method=method, resource_type=RESOURCE_VARIABLE, user=user)
 
     def is_authorized_view(self, *, access_view: AccessView, user: BaseUser | None = None) -> bool:
+        # Docs are just links the menu and not pages
```

Comment:
Can you expand/refine this just a little bit more? I'm a little confused what it means, I think it's missing just one or two words.

Reply:
Will do, sorry about that